dali/ft/filecopy.cpp

```diff
@@ -388,12 +387,13 @@ bool FileTransferThread::performTransfer()
         sprayer.tgtFormat.serializeExtra(msg, 1);
 
         ForEachItemIn(i2, progress)
-            progress.item(i2).serializeExtra(msg, 1);
+            progress.item(i2).serializeExtra(msg, 1); //must use version==1 to be compatible with old clients
 
         //NB: Any extra data must be appended at the end...
 
         msg.append(sprayer.fileUmask);
-
+        unsigned version = SUPPORTED_MSG_VERSION;
+        msg.append(version);
```

Comment:
I would expect a loop here calling 
serializeExtra(msg, 2);
for all the progress elements.  If not needed then there should be a comment why.